exchange/exchange.go

```diff
@@ -402,6 +410,40 @@ func (brw *bidResponseWrapper) validateBids() (err []error) {
 	return err
 }
 
+// validateCurrency will run currency validation checks and return true if it passes, false otherwise.
+func validateCurrency(requestAllowedCurrencies []string, bidCurrency string, defaultCurrency string) (bool, error) {
```

Comment:
Do we want to return both a boolean and an error? Currently we don't read the boolean value anywhere. If error is non-nil, we always return false, so I don't see what the boolean flag might gain us in future use.

Reply:
No, I agree we don't need this boolean, I'll remove it.